packages/grid/_modules_/grid/models/rows.ts

```diff
@@ -1,10 +1,9 @@
-export type RowsProp = RowData[];
-export type Rows = RowModel[];
+export type RowsProp = RowModel[];
 
 /**
  * The key value object representing the data of a row.
  */
-export interface RowData extends ObjectWithId {
+export interface RowModel extends ObjectWithId {
```

Comment:
I kept RowModel as we have a Row component so we avoid confusion

Reply:
It sounds good. I think that before the stable, we will need to name it `GridRowModel` to avoid conflict with other components.

Reply:
Let's do all the prefixes in a separate PR. Should we create an issue?

Reply:
#131